python/ray/_raylet.pxd

```diff
@@ -126,7 +126,7 @@ cdef class CoreWorker:
         object fd_to_cgname_dict
         object _task_id_to_future_lock
         dict _task_id_to_future
-        object thread_pool_for_async_event_loop
+        object event_loop_executor
```

Comment:
should we call it `thread_pool_executor_for_asnc_event_loop`?

Reply:
Not a big fan of adding sentences as names (`event_loop_executor` captures the intent perfectly)